File: CarlaChronicles-BackOnTrack/bot.cpp
#include <iostream>
#include <math.h>

float TAU = M_PI*2;

int main()
{
  float d = 100;
  float r = d/2;
  float circumference = TAU * r;
  std::cout << "Circumference: " << circumference << "\n";
}

File: CarlaChronicles-BackOnTrack/bot.go
package main

import (
  "strconv"
  "fmt"
  "math"
)

// Tau not Pi
const Tau = math.Pi * 2

func main() {
  var d float64 = 100
  var r = d/2
  circumference := Tau * r
  fmt.Println("Circumference: " + strconv.FormatFloat(circumference, 'f', -1, 64))
}
